src/sbt-test/sbt-autoversion/simple/test

```diff
@@ -0,0 +1,41 @@
+# Generate project files.
+> reload
+> compile
+
+# Setup repository.
+$ exec git init
+$ exec git add .
+$ exec git commit -m 'initial commit'
+$ exec git tag v0.0.0
+
+# Prove sbt can start.
+> reload
+> compile
+> show version
+
+# Make a bugfix commit.
+$ exec sed -i 's/CHANGE_ME/BUGFIX/' src/main/scala/Foo.scala
+$ exec git commit -am 'fix: rename foo'
+
+# Anticipate auto version.
+> reload
+> release with-defaults
+$ exec grep -Fq "0.0.2-SNAPSHOT" version.sbt
```

Comment:
Switched to using `grep` instead of `must-mirror` because `0.13`/`1.x` use different syntax for scoping the version. It also makes the scenario easier to read in-place :shrug: